open-codegen/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 Note: This is the Changelog file of `opengen` - the Python interface of OpEn
 
+## [0.6.10] - 2022-3-8
```

Comment:
1. Let's make this 2022-3-10 and release it tomorrow
2. There's no link for [0.6.10] - check out the end of the file (~ line 146)

Reply:
@smokinmirror